src/services/authentication/index.js

```diff
@@ -39,10 +39,18 @@ const authConfig = {
 
 class SCAuthenticationService extends AuthenticationService {
 	async getPayload(authResult, params) {
-		const payload = await super.getPayload(authResult, params);
-		const user = await this.app.service('usersModel').get(authResult.account.userId);
-		payload.schoolId = user.schoolId;
-		payload.roles = user.roles;
+		let payload = await super.getPayload(authResult, params);
+		if (authResult.account && authResult.account.userId) {
```

Comment:
And if not, some log? It`s look wrong if we can run in case with different payload.

Reply:
the authentication strategy can produce an account without userId (e.g. moodle/iserv if the user logs in for the first time), so it is expected behavior and not an error. I can add debug logs if you like.

Reply:
Debug for locally is nice to have, but not important. Because we must update the error handling in the next weeks.